src/utils/DialogMatches.ts

```diff
@@ -41,6 +41,8 @@ export const DialogMatches = {
     UpdateTextMsgSetupDialogMatch: /setup text message/i,
     NotifyDialogMatch: /notify/i,
     PopUpSignInDialogMatch: /signin/i,
+    AdaptiveCardDialog: /adaptive card/i,
+    AdaptiveCardDialogSubmitAction: "adaptive card",
```

Comment:
Remove this text from here and set it as a constant in the dialog itself.

Reply:
Done.